local.tf

```diff
@@ -1,19 +1,7 @@
 locals {
-  emr_cluster_name                = "aws-analytical-dataset-generator"
-  master_instance_type            = "m5.2xlarge"
-  core_instance_type              = "m5.2xlarge"
-  core_instance_count             = 1
-  task_instance_type              = "m5.2xlarge"
-  ebs_root_volume_size            = 100
-  ebs_config_size                 = 250
-  ebs_config_type                 = "gp2"
-  ebs_config_volumes_per_instance = 1
-  autoscaling_min_capacity        = 0
-  autoscaling_max_capacity        = 5
-  dks_port                        = 8443
-  dynamo_meta_name                = "DataGen-metadata"
-  hbase_root_path                 = format("s3://%s", data.terraform_remote_state.ingest.outputs.s3_buckets.hbase_rootdir)
-  secret_name                     = "/concourse/dataworks/adg"
+  emr_cluster_name = "aws-analytical-dataset-generator"
+  hbase_root_path  = format("s3://%s", data.terraform_remote_state.ingest.outputs.s3_buckets.hbase_rootdir)
```

Comment:
we may not this , looks like this wasn't removed when we moved away from hbase replica approach

Reply:
They are still passed in to the setup script so just left them for now